celesta-sql/src/main/java/ru/curs/celesta/score/Grain.java

```diff
@@ -473,14 +473,14 @@ public Namespace getNamespace() {
         }
 
         if (grainParts.isEmpty()) {
-            return null;
+            return Namespace.DEFAULT;
         }
 
         Iterator<GrainPart> i = grainParts.iterator();
         Namespace ns = i.next().getNamespace();
         while (i.hasNext()) {
-            if ((ns == null) || !ns.equals(i.next().getNamespace())) {
-                return null;
+            if (ns.equals(Namespace.DEFAULT) || !ns.equals(i.next().getNamespace())) {
```

Comment:
`ns.equals(Namespace.DEFAULT)` is generally not null-safe, it's always better to write `Namespace.DEFAULT.equals(ns)` or even `ns == Namespace.DEFAULT`.

Reply:
Done: `ns.equals(Namespace.DEFAULT)` -> `Namespace.DEFAULT.equals(ns)`